### PR TITLE
feat(customTemplate): create flipping card

### PR DESCRIPTION
**Pull request by simboonlong**
add css-loader, node-sass, sass-loader, style-loader
add scss loading into storybook webpack config
flip card on click

### CODE REVIEW

File: .storybook/webpack.config.js [hunk 1]
```diff
@@ -19,6 +21,14 @@ module.exports = ({ config }) => {
     test: /\.(ts|tsx)$/,
     use: [require.resolve("babel-loader"), require.resolve("react-docgen-typescript-loader")]
   });
+
+  // loader for scss files
+  config.module.rules.push({
```
Comment (Nebulis) [inline .storybook/webpack.config.js]: can we not use sass ?
- why do we need sass ?
- node-sass is pretty annoying (it's compiled on every machine ..)
- this project already uses some css lib (let's avoid to use too many things)

File: src/templates/customTemplate/customTemplate.tsx [hunk 2]
```diff
@@ -48,59 +97,69 @@ const Ratings: FunctionComponent<RatingsProps> = ({ ratings }: { ratings: Rating
...
 export const CustomTemplate: FunctionComponent<TemplateProps<CustomTemplateCertificate>> = ({ document }) => {
+  const [isFlipped, setFlip] = useState(false);
+
+  const onClickFlipCard = () => {
+    if (isFlipped) {
+      setFlip(false);
+    } else {
+      setFlip(true);
+    }
+  };
```
Comment (sebastianquek) [inline src/templates/customTemplate/customTemplate.tsx]: We can do `setFlip(prevFlip => !prevFlip)` instead

**Review by sebastianquek — COMMENTED** [review]
## General comments
- Good use of CSS transforms!
- The flipping interaction is not obvious at the moment, shall we add a button to indicate that the card can be flipped?

## Technical comments
- I noticed you used emotion's `css(...)`, inline css (`<div css={...}>`) and also `className` (which mostly uses bootstrap's classes). The styles are therefore scattered across several places which makes it quite hard to reason about. It's also challenging to logically link the styles with its corresponding use cases.
- I suggest we minimise the approaches for styling. I prefer using Emotion's `styled` approach (https://emotion.sh/docs/styled). This keeps the markup relatively clean and we treat each styled component like any other react component that accepts props.
```
const FlipCardFace = styled.div<{isFront: boolean}>`
  transform: rotateY(${({isFront}) => isFront ? '0deg' : '180deg'});
  ...
`;
...
<FlipCardFace isFront={true}>
```
- The styles are therefore more focused on the component level so we can look through the styles and understand quickly its use cases. This is opposed to specific properties that doesn't express where it's to be used. (e.g. `borderLeft`, though if we want to share styles, we can still fallback to this). 

Let me know what you think!

**Review by sebastianquek — APPROVED** [review]
👍

**Pull request merged**